utils/fetchEarthquakes.ts

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
 import { Earthquakes, EarthquakeFeature } from '@/types';
 
+//TODO: Why are we using axios? I could probably swap this for fetch and get some benefits from next
```

Comment:
I think I made this decision purely out of familiarity, but I can definitely swap it.